src/settings/MappingProfiles/detailsSections/hooks/useDisabledOrderFields.js

```diff
@@ -0,0 +1,33 @@
+import { useMemo } from 'react';
+
+import { useFieldMappingFieldValue } from './useFieldMappingFieldValue';
+
+import {
+  ORDER_FORMATS,
+  PO_STATUS
+} from '../constants';
+import {
+  ORDER_FORMAT_FILED,
+  PO_STATUS_FIELD
+} from '../../../../utils';
+
+/**
+ *
+ * @returns {{dismissElectronicDetails: boolean, dismissCreateInventory: boolean, dismissPhysicalDetails: boolean}}
+ * an object with indicators of disabled fields
+ */
+export const useDisabledOrderFields = () => {
+  const [poStatus, orderFormat] = useFieldMappingFieldValue([PO_STATUS_FIELD, ORDER_FORMAT_FILED]);
+
+  const dismissCreateInventory = useMemo(() => poStatus === PO_STATUS.OPEN, [poStatus]);
```

Comment:
I sure that using useMemo on lines 22,23,24 costs more then optimization it provides.  May we remove it?

https://react.dev/reference/react/useMemo#should-you-add-usememo-everywhere